shopify_app.gemspec

```diff
@@ -8,9 +8,7 @@ Gem::Specification.new do |s|
   s.author      = "Shopify"
   s.summary     = %q{This gem is used to get quickly started with the Shopify API}
 
-  s.add_dependency('rails', '>= 4.2.6', '< 5.0')
```

Comment:
@swalkinshaw , BTW, Gemfiles may be used during gem build and publishing procedure, which may or may not be a relevant factor for making decision here (and which, I believe, caused problem with currently published version of shopify_api gem)